config/webpack-config.js

```diff
@@ -58,102 +53,76 @@ const COMMON_UGLIFY_CONFIG = new UglifyWebpackPlugin( {
-  cache: true,
-  context: path.join( __dirname, '/../', paths.unprocessed, JS_ROUTES_PATH ),
-  entry: scriptsManifest.getDirectoryMap( paths.unprocessed + JS_ROUTES_PATH ),
+
+const COMMON_CHUNK_CONFIG = new webpack.optimize.CommonsChunkPlugin( {
+  name: COMMON_BUNDLE_NAME
+} );
+
+
+const commonConf = {
   module: COMMON_MODULE_CONFIG,
   output: {
-    path: path.join( __dirname, 'js' ),
     filename: '[name]'
   },
   plugins: [
-    new webpack.optimize.CommonsChunkPlugin( {
-      name: COMMON_BUNDLE_NAME
-    } ),
     COMMON_UGLIFY_CONFIG
   ]
 };
 
-const ieConf = {
-  entry: paths.unprocessed + '/js/ie/common.ie.js',
-  module: COMMON_MODULE_CONFIG,
-  output: {
-    filename: 'common.ie.js'
-  },
-  plugins: [ COMMON_UGLIFY_CONFIG ]
-};
-
 const externalConf = {
-  entry: paths.unprocessed + JS_ROUTES_PATH + '/external-site/index.js',
   module: COMMON_MODULE_CONFIG,
   output: {
     filename: 'external-site.js'
   },
-  plugins: [ COMMON_UGLIFY_CONFIG ]
+  plugins: [
+    COMMON_UGLIFY_CONFIG
+  ]
 };
 
-const onDemandConf = {
-  context: path.join( __dirname, '/../', paths.unprocessed,
-                      JS_ROUTES_PATH + '/on-demand' ),
-  entry:   scriptsManifest.getDirectoryMap( paths.unprocessed +
-                                            JS_ROUTES_PATH + '/on-demand' ),
+const modernConf = {
+  cache: true,
   module: COMMON_MODULE_CONFIG,
   output: {
-    path:     path.join( __dirname, 'js' ),
     filename: '[name]'
   },
-  plugins: [ COMMON_UGLIFY_CONFIG ]
+  plugins: [
+    COMMON_CHUNK_CONFIG,
+    COMMON_UGLIFY_CONFIG
+  ]
 };
 
 const onDemandHeaderRawConf = {
-  context: path.join( __dirname, '/../', paths.unprocessed,
-                      JS_ROUTES_PATH + '/on-demand' ),
-  entry:  './header.js',
-  module: COMMON_MODULE_CONFIG,
-  output: {
-    path:     path.join( __dirname, 'js' ),
-    filename: '[name]'
-  }
+  module: COMMON_MODULE_CONFIG
 };
 
-const spanishConf = {
-  entry: paths.unprocessed +
-         JS_ROUTES_PATH + '/es/obtener-respuestas/single.js',
+const owningAHomeConf = {
+  cache: true,
   module: COMMON_MODULE_CONFIG,
   output: {
-    filename: 'spanish.js'
+    filename: '[name]',
+    jsonpFunction: 'OAH'
   },
-  plugins: [ COMMON_UGLIFY_CONFIG ]
+  plugins: [
+    COMMON_CHUNK_CONFIG,
+    COMMON_UGLIFY_CONFIG
+  ]
 };
 
-const owningAHomeConf = {
-  cache: true,
-  context: path.join( __dirname, '/../',
-    paths.unprocessed, OAH_COMMON_BUNDLE_ROUTE
-  ),
-  entry: scriptsManifest.getDirectoryMap(
-    paths.unprocessed + OAH_COMMON_BUNDLE_ROUTE
-  ),
+const spanishConf = {
   module: COMMON_MODULE_CONFIG,
   output: {
-    path: path.join( __dirname, 'js' ),
-    filename: '[name]',
-    jsonpFunction: 'OAH'
+    filename: 'spanish.js'
   },
   plugins: [
-    new webpack.optimize.CommonsChunkPlugin( {
-      name: COMMON_BUNDLE_NAME
-    } ),
-    COMMON_UGLIFY_CONFIG ]
+    COMMON_UGLIFY_CONFIG
+  ]
 };
 
 module.exports = {
+  commonConf:            commonConf,
```

Comment:
you can just use shorthand for these:
`{ commonConf,
    externalConf,
   etc..
}`.